Translation-French.md

```diff
@@ -6,6 +6,7 @@ Français ·
 [Русский](./Translation-Russian.md) ·
 [Deutsch](./Translation-German.md) ·
 [日本語](./Translation-Japanese.md)
```

Comment:
Add the separator, please (in all translations).

Reply:
Done